app/src/main/java/zafrani/com/pubgapp/models/Category.java

```diff
@@ -2,25 +2,49 @@
 
 
 import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
 
 import com.google.gson.annotations.SerializedName;
 
+import java.util.List;
+
 public class Category {
     @SerializedName("cat_name")
-    private String name;
+    private String cat_name;
+
+    @Nullable
+    @SerializedName("types")
+    private List<Type> types;
```

Comment:
certain Categories like ammunition didnt have a type field, i changed the json in this or the itemlist pull request to be consistent so it doesnt have to be nullable.